DOCKER.md

```diff
@@ -0,0 +1,41 @@
+# Using Docker to run OpenStreetMap
+
+Using [Docker](https://www.docker.com/) will allow you to install the OpenStreetMap application and all it's dependencies in a container.
+
+### Clone the repo
+
+```
+git clone --depth=1 https://github.com/openstreetmap/openstreetmap-website.git
+```
+
+### App configuration
+
+```
+cp config/example.application.yml config/application.yml
+```
+
+### Database
+
+```
+cp config/example.database.yml config/database.yml
+```
+
+Set `username` to postgres and `host` to db leave the password blank
```

Comment:
The guides I've read keep this kind of configuration separate. I would suggest leaving it out for now and it could be reviewed later.